app/templates/index.html

```diff
@@ -9,31 +9,30 @@
 
 
 {% block main %}
-<section class="tagline">
-    <h1>All things tech at Columbia</h1>
-</section>
-<section class="card top-card">
-    <div class="content">
-
-        <form action="//adicu.us4.list-manage.com/subscribe/post?u=62d8b690f349b99dd7e0f3205&amp;id=01a495e6ed"
-              method="post"
-              id="gi-embedded-subscribe-form"
-              name="gi-embedded-subscribe-form"
-              class="subscribe validate"
-              target="_blank">
-            <div class="left">
-                <input type="email"
-                       name="EMAIL"
-                       id="gi-EMAIL"
-                       placeholder="Subscribe to our newsletter..."
-                       required />
-            </div>
-            <div class="button submit">Sign Up!</div>
-            <input class="hidden"
-                   type="submit"
-                   id="gi-embedded-subscribe"
-                   name="subscribe">
-        </form>
+<section class="newsletter">
+    <div class="card top-card">
+        <div class="content">
+            <h1 class="page-title">All things tech at Columbia</h1>
+            <form action="//adicu.us4.list-manage.com/subscribe/post?u=62d8b690f349b99dd7e0f3205&amp;id=01a495e6ed"
+                  method="post"
+                  id="gi-embedded-subscribe-form"
+                  name="gi-embedded-subscribe-form"
+                  class="subscribe validate"
+                  target="_blank">
+                <div class="left">
+                    <input type="email"
+                           name="EMAIL"
+                           id="gi-EMAIL"
+                           placeholder="Subscribe to our newsletter..."
+                           required />
```

Comment:
It's perfectly valid:http://stackoverflow.com/questions/7366344/do-we-still-need-end-slashes-in-html5

But we should remove it, for style reasons